source/client/hud.qc

```diff
@@ -1934,6 +1934,56 @@ void(float width, float height) HUD_RoundStopWatch =
 	Draw_String([width - (getTextWidth(stopwatch, 12)) - 2, 16], stopwatch, [12, 12], [1, 1, 1], 1, 0);
 }
 
+/*******************
+*  HUD Debug Info  *
+*******************/
+
+// set from CSQC_Ent_Update
+vector player_velocity;
+
+// probably only need these right here
+static void(vector pos, string label, vector v) Draw_Vector_Fancy =
+{
+	Draw_String(pos, label, [12, 12], [1, 1, 0], 1, 0);
+	Draw_String(pos + [32, 16], sprintf("X: %d", v.x), [12, 12], [1, 1, 1], 1, 0);
+	Draw_String(pos + [32, 32], sprintf("Y: %d", v.y), [12, 12], [1, 1, 1], 1, 0);
+	Draw_String(pos + [32, 48], sprintf("Z: %d", v.z), [12, 12], [1, 1, 1], 1, 0);
+};
+
+static void(vector pos, string label, float f, string suffix) Draw_Float_Fancy =
+{
+	Draw_String(pos, label, [12, 12], [1, 1, 0], 1, 0);
+	Draw_String(pos + [32, 16], sprintf("%d %s", f, suffix), [12, 12], [1, 1, 1], 1, 0);
+};
```

Comment:
Can we move these to `source/client/draw.qc`? Just in case we might actually use them elsewhere, keeps things tidy and future proof. `Draw_FancyFloat` and `Draw_FancyVector` might also be names that better fit the style here, but that's a nitpick, so feel free to ignore